core/src/main/java/io/varhttp/VarHttpServletRequest.java

```diff
@@ -205,7 +202,7 @@ public String getMethod() {
 
 	@Override
 	public ServletInputStream getInputStream() throws IOException {
-		return is;
+		return new WrappedServletInputStream(is);
```

Comment:
should it not be wrapped in constructor?

Reply:
This is fixed in the merge branch, see other pr